pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerGauge.java

```diff
@@ -43,6 +43,7 @@
   LLC_SIMULTANEOUS_SEGMENT_BUILDS("llcSimultaneousSegmentBuilds", true),
 
   // Upsert metrics
+  UPSERT_METADATA_MEMORY_USED("bytes", false),
```

Comment:
UPSERT_METADATA_MEMORY_USED("upsertMetadataMemoryUsed", false),

Reply:
Gotcha, thanks for pointing out!